Upstream: Should be
Reason: Fixes compilation with gcc-10

Taken from https://src.fedoraproject.org/rpms/sysfsutils/blob/master/f/0002-fix-compiler-complaints.patch

diff --git a/test/test-defs.h b/test/test-defs.h
index b22909a..73ede6d 100644
--- a/test/test-defs.h
+++ b/test/test-defs.h
@@ -42,7 +42,7 @@
 #define inval_path 		"/sys/invalid/path"
 #define FUNC_TABLE_SIZE  	(sizeof(func_table)/sizeof(int))
 
-FILE *my_stdout;
+extern FILE *my_stdout;
 
 #define dbg_print(format, arg...) fprintf(my_stdout, format, ## arg)
 
diff --git a/test/test.c b/test/test.c
index 2e8f201..f63e346 100644
--- a/test/test.c
+++ b/test/test.c
@@ -27,6 +27,8 @@
 #include "test-defs.h"
 #include <errno.h>
 
+FILE *my_stdout;
+
 /*************************************************/
 char *function_name[] = {
 	"sysfs_get_mnt_path",
